sql/dolphinscheduler_mysql.sql

```diff
@@ -968,3 +968,16 @@ CREATE TABLE `t_ds_alert_plugin_instance` (
   `instance_name` varchar(200) DEFAULT NULL COMMENT 'alert instance name',
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- ----------------------------
+-- Table structure for t_ds_alert_plugin_instance
+-- ----------------------------
+DROP TABLE IF EXISTS `t_ds_audit_log`;
+CREATE TABLE `t_ds_audit_log` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `user_id` int(11) NOT NULL COMMENT 'user id',
+  `module` int DEFAULT NULL COMMENT 'module',
+  `operation` int DEFAULT NULL COMMENT 'operation',
+  `time` datetime DEFAULT NULL COMMENT 'create time',
```

Comment:
```suggestion
`time` datetime DEFAULT CURRENT_TIMESTAMP COMMENT 'create time',
```

Reply:
@ruanwenjun  Thanks for your review, I have addressed the comments. PTAL